images/app/helpers/refinery/admin/images_helper.rb

```diff
@@ -19,6 +19,12 @@ def thumbnail_urls(image)
 
         thumbnail_urls
       end
+
+      # We show the title from the next available locale
+      # if there is no title for the current locale
```

Comment:
So, this method is not good too : https://github.com/refinery/refinerycms/blob/master/pages/app/helpers/refinery/admin/pages_helper.rb#L50 ?